test/module/shared_model/builders/protobuf/builder_templates/proposal_template.hpp

```diff
@@ -61,7 +61,8 @@ namespace shared_model {
       // we do such default initialization only because it is deprecated and
       // used only in tests
       TemplateProposalBuilder()
-          : TemplateProposalBuilder(SV(iroha::test::kTestsValidatorsConfig)) {}
+          : TemplateProposalBuilder(
+                SV(iroha::test::kProposalTestsValidatorsConfig)) {}
```

Comment:
Is it required to create proposals with duplicate transactions with the builder? If not, then the change can be reverted.

Reply:
I added a flag for choosing config in the constructor